proto/kvrpcpb.proto

```diff
@@ -815,6 +814,9 @@ message ScanDetailV2 {
 
     // Total number of bytes from block reads.
     uint64 rocksdb_block_read_byte = 7;
+
+    // Total number of bytes from Scan or kvGet
```

Comment:
The comment does not match the field name, how about changing it to "Total number of bytes read from RocksDB"?

BTW what is the behaviour when it comes to tombstones or deleted (but not gced) multi version keys?

Reply:
Done